code/datums/action.dm

```diff
@@ -402,12 +404,13 @@
 /datum/action/item_action/ninja_stealth
 	name = "Toggle Stealth"
 	desc = "Toggles stealth mode on and off."
-	button_icon_state = "beam_up"
+	button_icon_state = "ninja_cloak"
```

Comment:
as note, ninja_cloak and ninja_jaunt should both probably not be animated, as it's distracting.